src/components/atoms/Modal/Modal.component.tsx

```diff
@@ -7,11 +7,7 @@ import { ThemeProvider } from '@src/styles/theme'
 
 import { Box } from '../Box/Box.component'
 
-export enum ModalPositionEnum {
-  DEFAULT = 'default',
-  TOP = 'top',
-  CENTER = 'center'
-}
+export type ModalOptionsPosition = 'default' | 'top' | 'center'
```

Comment:
use can do something like
`${ModalPositionEnum}` it will convert to 'default' | 'top' | 'center'